content/en/user-guide/aws/docdb/index.md

```diff
@@ -231,16 +232,125 @@ the [MongoDB documentation](https://www.mongodb.com/docs/).
 ### Connect to DocumentDB using Node.js Lambda
```

Comment:
It would be very helpful (for us) to link to the scenario test you mentioned here: https://github.com/localstack/localstack-ext/blob/master/tests/aws/scenario/rds_neptune_docdb/test_rds_neptune_docdb.py#L112
(ideally vice-versa)

Even if it's just an internal comment (not visible in the docs).

Reply:
only if we don't move it 😅 
stupid question: is there something in hugo that we can use to add comments, that will not be rendered/showing up in the docs? 🤔

Reply:
found something

Reply:
HTML or Go template comments should do it: https://gohugo.io/templates/introduction/#comments